pontoon/base/views.py

```diff
@@ -485,6 +486,28 @@ def delete_translation(request):
     return JsonResponse({"status": True})
 
 
+@utils.require_AJAX
+@login_required(redirect_field_name="", login_url="/403")
+@transaction.atomic
+def delete_comment(request):
+    """Delete given comment."""
+    try:
+        c = request.POST["comment"]
+    except MultiValueDictKeyError as e:
```

Comment:
I don't think this can happen from the previous line. You probably just want to check for a missing value here, or an invalid integer.

Reply:
Will that change if we add the `created-at` parameter as discussed for the action log?